script.js

```diff
@@ -1,60 +1,75 @@
+    this.bookTitle = document.querySelector('#Title');
+    this.bookAuthor = document.querySelector('#Author');
+    this.bookLibrary = [];
+    this.addBook = this.addBook.bind(this);
+  }
 
-if (localStorage.getItem('books') == null) {
-  localStorage.setItem('books', JSON.stringify(bookLibrary));
-}
+  // displays already added books to storage
+  storedLocal() {
+    if (localStorage.getItem('books') == null) {
+      localStorage.setItem('books', JSON.stringify(this.bookLibrary));
+    }
 
-const store = localStorage.getItem('books');
-if (store) {
-  bookLibrary = JSON.parse(store);
-}
+    const store = localStorage.getItem('books');
+    if (store) {
+      this.bookLibrary = JSON.parse(store);
+    }
+  }
 
-function createLibrary() {
-  BooksContainer.innerHTML = '';
-  bookLibrary.forEach((book, index) => {
-    const bookContainer = document.createElement('div');
-    bookContainer.innerHTML = `
-        <p>${book.title}</p>
-       <p> ${book.author}</p>
+  // create a new book display
+  createLibrary() {
+    this.BooksContainer.innerHTML = '';
+    this.bookLibrary.forEach((book, index) => {
+      const bookContainer = document.createElement('div');
+      bookContainer.innerHTML = `
+      <div class='book-list-container'>
+        <div class='wrap'>
+          <div>
+              <p>${book.titleBook}</p>
+              <p> ${book.authorBook}</p>
+          </div>
+          <div class='removeBtn'>
             <button onclick="removeBook(${index})">Remove</button>
-        <hr>
-    `;
-    BooksContainer.appendChild(bookContainer);
-  });
-}
+          </div>
+        </div>
+      </div>
+      `;
+      this.BooksContainer.appendChild(bookContainer);
+    });
+  }
 
-// Add New Book to The List
-function addBook(e) {
-  e.preventDefault();
-  const newBooks = {
-    title: bookTitle.value,
-    author: bookAuthor.value,
-  };
+  // take in and stores new book in local storage
+  addBook(e) {
+    e.preventDefault();
+    const titleBook = this.bookTitle.value;
+    const authorBook = this.bookAuthor.value;
+    const newBooks = { titleBook, authorBook };
 
-  bookLibrary.push(newBooks);
-  localStorage.setItem('books', JSON.stringify(bookLibrary));
+    this.bookLibrary.push(newBooks);
+    localStorage.setItem('books', JSON.stringify(this.bookLibrary));
 
-  createLibrary();
+    this.createLibrary();
 
-  // Reset input fields
-  bookTitle.value = '';
-  bookAuthor.value = '';
+    this.bookTitle.value = '';
+    this.bookAuthor.value = '';
+  }
 }
 
-// Add New Book to The List
+const objectOne = new Bookstore();
 
 function removeBook(index) {
-  bookLibrary = bookLibrary.filter((book, bookIndex) => bookIndex !== index);
-  localStorage.setItem('books', JSON.stringify(bookLibrary));
+  objectOne.bookLibrary = objectOne.bookLibrary.filter((book, bookIndex) => bookIndex !== index);
+  localStorage.setItem('books', JSON.stringify(objectOne.bookLibrary));
 
-  createLibrary();
+  objectOne.createLibrary();
 }
```

Comment:
- [ ] I think there's some confusion because as previously mentioned you should move the definition of the `removeBook()` function inside of the class just like the `addBook()` is inside the class. This is part of the project requirements so please carefully implement it. ✔